src/Response/Model/DirectReactions.php

```
@@ -0,0 +1,15 @@
+<?php
+
+namespace InstagramAPI\Response\Model;
+
+/**
+ * @method int getLikesCount()
+ * @method null|DirectReaction[] getLikes()
```

Actually we are open for _any_ class. For example, someone can call `$instagram->request('/some/endpoint/')->getResponse(new \stdClass());` and we will fail hard. So `ResponseInterface` is kind of necessary thing to add.

As for traits, they will only help with those 2 (now 3) models used as raw responses.